src/docs/implementations/dynatrace.adoc

```diff
@@ -37,7 +37,16 @@ DynatraceConfig dynatraceConfig = new DynatraceConfig() {
 MeterRegistry registry = new DynatraceMeterRegistry(dynatraceConfig, Clock.SYSTEM);
 ----
 
-It is also possible to specify the exporter version explicitly, which defaults to `v2` unless a deviceId is set:
+If you are using Micrometer 1.10.0 or above, you can also use the DEFAULT config to achieve the same with less code:
+
+[source,java]
+----
+DynatraceConfig dynatraceConfig = DynatraceConfig.DEFAULT;
+MeterRegistry registry = new DynatraceMeterRegistry(dynatraceConfig, Clock.SYSTEM);
```

Comment:
You could make it even more concise:
```suggestion
MeterRegistry registry = new DynatraceMeterRegistry(DynatraceConfig.DEFAULT, Clock.SYSTEM);
```

Reply:
I think this is a good suggestion, @pirgeo could you please apply it if you agree?